configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,931 @@ benchmark_configs: {
       "}"
   }
 }
+benchmark_configs: {
+  id: "audit-backlog-limit-sufficient"
+  compliance_note: {
+    title: "Ensure Audit Backlog Limit is Sufficient"
+    description: 
+      "This configuration ensures that the audit subsystem's queue for audit events has a sufficient backlog limit to "
+      "prevent loss of audit data during high activity periods."
+    rationale: 
+      "Setting a sufficient audit backlog limit minimizes the risk of losing critical audit data, enhancing the reliability "
+      "of audit records in detecting potential malicious activities."
+    remediation: 
+      "Edit `/etc/default/grub` to include `audit_backlog_limit=8192` or a higher value in the `GRUB_CMDLINE_LINUX` "
+      "configuration. Update the GRUB configuration with the command `update-grub`."
+    cis_benchmark: {
+      profile_level: 2
+      severity: HIGH
+    }
+    scan_instructions:
+      "scan_type_specific:{instance_scanning:{"
```

Comment:
There's no image_scanning check defined for this. Either replace this with "generic" or add a separate image_scanning check too.